spec/src/main/asciidoc/architecture.adoc

```diff
@@ -196,12 +196,15 @@ In the above examples both `countMeA()` and `countMeB()` will share a single Cou
 
 Data is exposed via REST over HTTP under the `/metrics` base path in two different data formats for `GET` requests:
 
-* JSON format - used when the HTTP Accept header matches `application/json`.
-* OpenMetrics text format - default response format when the HTTP Accept header does not match any more specific
-media type like `application/json`.
+* JSON format - used when the HTTP Accept header best matches `application/json`.
+* OpenMetrics text format - used when the HTTP Accept header best matches `text/plain` or when Accept header would equally
+accept both `text/plain` and `application/json` and there is no other higher precedence format.
 +
-NOTE: Future versions may allow for more export formats that are triggered by their specific media type. The OpenMetrics
-text format will stay as fall-back.
+This format is also returned when no media type is requested (i.E. no Accept header is provided in the request)
```

Comment:
The spacing between "no other higher precedence format." and "This format is..." makes it hard to tell that the latter is just a continuation of the text/plain case.  Perhaps just add "This format is ..." as a sentence right after "...precedence format."

Reply:
Took me a while to get what you meant, but yes you are right, it is better with direct continuation.